deal.II-toolchain/packages/symengine.package

```diff
@@ -0,0 +1,30 @@
+VERSION=0.4.0
+CHECKSUM=fd9ae3e98258291ef35b6392faa72ace
```

Comment:
We recently installed symengine successfully with version 0.7.0
VERSION=0.7.0
CHECKSUM=72b8f1442fd3084c96e59d9fe87eec1a